i18n/en/docusaurus-plugin-content-docs/current/intro.mdx

```diff
@@ -23,18 +23,41 @@ FSD doesn't enforce a particular programming language, UI framework or state man
 
 If you have an existing project, fear not — FSD can be adopted incrementally. Just make sure that your team is **in&nbsp;pain** from the current architecture, otherwise a switch might not be worth it. <!-- For migration guidance, see the Migration section. TODO: add the migration section and link to it here -->
 
-## Basics
+## Overview
 
-In radically simplified terms, FSD boils down to the following:
-
-- Project partitioning on business domains
-- Specific and consistent file structure (see the image below)
+In FSD, a project consists of _layers_, each layer is made up of _slices_ and each slice is made up of _segments_. 
 
 ![themed--scheme](/img/visual_schema.jpg)
 
-The terms in the diagram above are explained in detail in the [**Abstractions**][refs-basics--abstractions] section.
+The layers are standardized across all projects and vertically arranged. Modules on one layer can only interact with modules from the layers strictly below. There are currently seven of them (bottom to top):
+
+1. **Shared** — reused functionality, detached from the specifics of the project/business.
```

Comment:
I meant it in this way specifically to communicate the idea that people shouldn't throw stuff into shared if they only use it once, just because they think it might be useful. At least that's what I think the best practice would be

Reply:
The main criterion for shared code is being abstract ( no business logic ), so it gives you reusability, but you **don't have to** use it until you really need this

The similar logic works for features – it's reasonable to isolate dedicated business logic aspect even if it's needed on only one page